index.html

```diff
@@ -127,122 +127,11 @@ <h3 class="multi-post">Multi-Post Stories</h3>
             </ul>
             <button class="see-project1">See Project</button>
           </div>
```

Comment:
- [ ] 🙏Please implement the popup window details according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). Currently, it does not even show the "See Project" button in desktop size. This will allow viewers👨‍👩‍👧‍👦 to see the details of your projects in your portfolio.